docs/dunst.5.pod

```diff
@@ -58,6 +58,18 @@ keyboard focus.
 
 =back
 
+=item B<enable_posix_regex> (default: false)
+
+When set to true, you can use POSIX regular expressions for filtering rules. It
+uses the POSIX Extended Regular Expression syntax:
+https://en.m.wikibooks.org/wiki/Regular_Expressions/POSIX-Extended_Regular_Expressions.
+
+If this is set to false, dunst will us fnmatch(3) for matching strings.
```

Comment:
_nit_: _use_

Might be worth noting that `dunst` passes no (?) extra flags to `fnmatch` as well?

Reply:
Thanks, I'll add both your suggestions.